packages/release/activate-configured.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Isolate configured.target
+After=preconfigured.target
+
+[Service]
+Type=oneshot
+ExecStart=systemctl isolate configured
```

We should use the full path to `systemctl`, so we don't depend on any implicit `$PATH` setting:
```suggestion
ExecStart=/usr/bin/systemctl isolate configured
```